charts/jenkins/values.yaml

```diff
@@ -169,6 +169,15 @@ controller:
       # It delays the initial readyness probe while Jenkins is starting
       # initialDelaySeconds: 60
 
+  # PodDisruptionBudget config
+  podDisruptionBudget:
+    enabled: false
+    annotations: {}
+    labels: {}
+    # Default to not allowing any unavailable nodes
+    # This ensures that Jenkins is available as much as possible
+    maxUnavailable: "0"
```

Comment:
bit risky with things like kured this will prevent patching from occurring on a machine?

Reply:
Yeh, so it'll prevent the pod from being killed without operator intervention...
Which in our use case is acceptable, but others opinions might differ I guess... 

Alternatively can remove the default so that it's up to the user to specify what they want...

Reply:
I thik it would be better if the user actively decides to handle it this way.

Reply:
Default value removed in [`fa94718` (#461)](https://github.com/jenkinsci/helm-charts/pull/461/commits/fa94718414a334addc6e6fd39d80a9d2170317c4)